airbyte/datasets/_base.py

```diff
@@ -37,6 +37,13 @@ def to_pandas(self) -> DataFrame:
         # duck typing is correct for this use case.
         return DataFrame(cast(Iterator[dict[str, Any]], self))
 
+    def to_arrow_dataset(self, chunksize: int = 100000) -> Any:
```

Comment:
What should be optimal chunk_size by default?

Reply:
10_000 or 100_000 feels like a decent place to start here. Maybe 100_000? And what do you think of adding a constant at the top of the file to hold this default?

Something like:

```py
DEFAULT_ARROW_CHUNKSIZE = 10)_000
```

And then the inline declaration here would be:

```suggestion
    def to_arrow_dataset(self, chunksize: int = DEFAULT_ARROW_CHUNKSIZE) -> Any:
```

Reply:
10_000 seems nice and added variable